### PR TITLE
Issue template update effort estimation

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

- Added dropdown reminder for effort estimation
- Changed "Specific Project Types (EE)" and "Specific Team Tiers (EE)" to "Licensed Edition (EE)"

## Related PR

https://github.com/flowforge/website/pull/685

### CODE REVIEW

**Review by ZJvandeWeg — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/01-bug_report.yml [hunk 1]
```diff
@@ -32,3 +32,15 @@ body:
...
     required: false
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/01-bug_report.yml:43]:
```suggestion
      - I am not a FlowForge team member
```

File: .github/ISSUE_TEMPLATE/03-feature.yml [hunk 2]
```diff
@@ -6,3 +6,15 @@ body:
...
     description: Describe the feature
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/03-feature.yml:17]:
```suggestion
      - I am not a FlowForge team member
```

File: .github/ISSUE_TEMPLATE/05-epic.yml [hunk 3]
```diff
@@ -12,3 +12,16 @@ body:
...
     description: Describe the epic
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/05-epic.yml:23]:
```suggestion
      - I am not a FlowForge team member
```

File: .github/ISSUE_TEMPLATE/05-epic.yml [hunk 4]
```diff
@@ -12,3 +12,16 @@ body:
...
+      - I can not provide an initial effort estimate
+      - All subtasks of this epic have an effort estimate
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/05-epic.yml:25]: At the point in time the epic is raised we won't expect to have all the subtasks already raised and sized. So this option will never be used when raising the epic. Once raised, I'm not sure if the template carries over to be edited later...

File: .github/ISSUE_TEMPLATE/06-story.yml [hunk 5]
```diff
@@ -46,3 +45,16 @@ body:
...
+
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/06-story.yml:57]:
```suggestion
      - I am not a FlowForge team member
```

File: .github/ISSUE_TEMPLATE/07-task.yml [hunk 6]
```diff
@@ -16,3 +16,15 @@ body:
...
     placeholder: '#123'
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
```
Comment (knolleary) [inline .github/ISSUE_TEMPLATE/07-task.yml:27]:
```suggestion
      - I am not a FlowForge team member
```

**Review by knolleary — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/05-epic.yml (hunk 4 above)
Comment (MarianRaphael) [reply .github/ISSUE_TEMPLATE/05-epic.yml:25]: Agree, I will change it to a hint.

Comment (MarianRaphael) [reply .github/ISSUE_TEMPLATE/05-epic.yml]: [changed dropdown to hint for epics](https://github.com/flowforge/flowforge/pull/2046/commits/3b45e184418ff1dd62761b9079ea224dc75d87e4)

**Review by knolleary — APPROVED** [review]

**Pull request merged**